.prettierignore

```diff
@@ -88,48 +95,62 @@ typings/
 .yarn-integrity
 
 # local environment variable files
+blocks/**/.env
 .env.local
 .env.*.local
 
 # macOS directory file
-.DS_Store
+**/.DS_Store
 
 # next.js build output
-.next
+**/.next
 
 # build files
-dist
-build
-target
+**/dist
+**/build
+**/target
 
 # vscode config
-.vscode/settings.json
+**/.vscode/settings.json
 
 # intellij IDEs config
-.idea/
+**/.idea/
 
 # fleet IDEs config
-.fleet/
+**/.fleet/
+
+# Python virtual environment
+**/venv
+**/.venv
 
 # Python compiled files and package metadata
-__pycache__
+**/__pycache__
 *.egg-info
+**/*.pyc
+**/*.pyo
+**/*.pyd
+**/.Python
+**/pip-log.txt
+**/pip-delete-this-directory.txt
+**/.mypy_cache
+**/.pytest_cache
+**/.hypothesis
 
 # generated files
 *.gen.*
-blocks/**/generated
 *.tsbuildinfo
+blocks/**/generated
```

Comment:
Ah hm, I don't think this will work as intended

Reply:
There are some unnecessary patterns (e.g. `**/minesweeper/src/**` is equivalent to `minesweeper/src/**`) but that should be fine

Reply:
Yep, but it would have been easy to accidentally add something like this in the future, so I wanted to check it behaves as expected (we don't want to ignore `/blocks/**/potato/src/**` either 😁 )